### PR TITLE
KAZOO-4529: simpler dev/staging split for releases` vm.args file

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by lazedo** [comment]
@fenollp does a simple `make` create the `dev-vm.args` ?

**Comment by fenollp** [comment]
a simple make used to create `dev-vm.args`. I'll make it do so again.

**Comment by lazedo** [comment]
@fenollp since you remove the app.config from vm.arg (renamed args), a simple make would not build the app.config or sys.config into dev-vm.args.

File: Makefile [hunk 1]
```diff
@@ -51,15 +50,21 @@ kazoo: core apps
...
 	$(RELX) --config $< -V 2 release --relname 'kazoo'
-tar-release: rel/relx.config
+tar-release: rel/relx.config rel/vm.args
 	$(RELX) --config $< -V 2 release tar --relname 'kazoo'
 rel/relx.config: rel/relx.config.src
 	$(ROOT)/scripts/src2any.escript $<
 
+rel/dev-vm.args: rel/args  # Used by scripts/dev-start-*.sh
+	cp $^ $@
+rel/vm.args: rel/args rel/dev-vm.args
```
Comment (lazedo) [inline Makefile]: why does rel/vm.args depend on dev-vm.args ?

Comment (fenollp) [inline Makefile]: because

**Pull request merged**